src/actions/clientActions.js

```diff
@@ -0,0 +1,22 @@
+import { CLIENT_CHANGE } from '../constants/actionTypes'
+
+function isSmallScreen () {
+  return global.window.innerWidth <= 768
+}
+
+function clientChange (smallScreen) {
+  return {
+    type: CLIENT_CHANGE,
```

Comment:
@MMajko, what do you think? Do you think we will have to track additional client features in the future?